test/e2e/lib/components/sidebar-component.js

```diff
@@ -64,10 +59,15 @@ export default class SidebarComponent extends AsyncBaseContainer {
 	}
 
 	async selectPlans() {
-		await this.expandDrawerItem( /^Plan\b/ );
+		await this.expandDrawerItem( 'Upgrades' );
 		return await this._scrollToAndClickMenuItem( 'plans' );
 	}
 
+	async selectDomains() {
+		await this.expandDrawerItem( 'Upgrades' );
+		return await this._scrollToAndClickMenuItem( 'domains' );
```

Comment:
This is perfectly fine (and it matches the other functions here), but just so _you_ know, there's no need for the `return await functionThatReturnsAPromise()` pattern here since this function is already marked `async` (it will always return a Promise) and therefore just `return functionThatReturnsAPromise()` will have the same effect.

Reply:
Documented better here: https://github.com/eslint/eslint/blob/master/docs/rules/no-return-await.md